### ISSUE TITLE
[dashboard] give better hints how to unsubscribe

### ISSUE THREAD

**Issue by JanKoehnlein** [gitpod-io/gitpod#4453]
We get quite a lot of emails of users that want to unsubscribe, in particular as responses to invoices. Our UX should be self-explanatory in this regard.

Proposals
1) On the plans page, make it clearer that you can switch back to the free plan an as such unsubscribe from any paid plan.
2) In the invoice mails, add a link to the plan page on the dashboard

Some UX input on this is weclome @gtsiolis

**Comment by gtsiolis** [comment]
A good improvement here could be to a) separating teams and individual plans which are currently mixed and b) surfacing a more clear acton button for managing individual subscription as described in https://github.com/gitpod-io/gitpod/issues/3948#issuecomment-869453517. Cc @JanKoehnlein

**Comment by JanKoehnlein** [comment]
Does it make sense to look into this when we'll release the new teams and projects soon anyway?

**Comment by gtsiolis** [comment]
> Does it make sense to look into this when we'll release the new teams and projects soon anyway?

@JanKoehnlein Yes! For the separation of the individual and team plans, I'd expect the new **Teams & Projects** feature to resolve most of the pain points here. However, surfacing a more clear action to manage the subscription could be a small nice improvement.
